index.html

```diff
@@ -1,21 +1,270 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="index.css" />
-    <title>Credard</title>
-  </head>
-  <body>
-    <!-- where the nav should be -->
-    <header class="landing-page_header">
-      <h1>
-        Say bye to boring credit cards, get your own
-        <span>exciting designs!</span>
-      </h1>
-    </header>
-    <main></main>
-    <footer></footer>
-  </body>
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="index.css" />
+        <title>Credard</title>
+    </head>
+    <body>
+        <!-- where the nav should be -->
+        <main>
+            <section class="hero">
+                <figure class="hero_fig">
+                    <div class="hero_img-c">
+                        <img
+                            class="hero_img"
+                            src="./assets/images/credit-cards-hero.svg"
+                            alt=""
+                        />
+                    </div>
+                    <figcaption class="hero_figc">
+                        <h1>
+                            Say bye to boring credit cards, get your own
```

Comment:
Text too big on mobile, also add navigation to the page
![image](https://user-images.githubusercontent.com/60205816/183245499-004dd4cb-9f1e-4cbd-8f40-8a747400e4bd.png)